### PR TITLE
Add initial releasing instructions.

### PR DESCRIPTION
**Pull request by bogdandrutu**

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+1.  Create the release branch and push it to GitHub:
+
+    ```bash
+    $ MAJOR=0 MINOR=4 PATCH=0 # Set appropriately for new release
+    $ VERSION_FILES=(
+      build.gradle
+      )
+    $ git checkout -b v$MAJOR.$MINOR.x master
+    $ git push upstream v$MAJOR.$MINOR.x
+    ```
```
Comment (sebright) [inline RELEASING.md]: Is "upstream" the main repository or a personal fork?

File: RELEASING.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    You'll also need to [publish your public
+    key](http://central.sonatype.org/pages/working-with-pgp-signatures.html#distributing-your-public-key)
+    to make it visible to the Sonatype servers.
+-   Put your GnuPG key password and OSSRH account information in
+    `<your-home-directory>/.gradle/gradle.properties`.
+
+```
+# You need the signing properties only if you are making release deployment
+signing.keyId=<8-character-public-key-id>
+signing.password=<key-password>
```
Comment (bogdandrutu) [inline RELEASING.md]: Yes it is. Tried to make it more obvious.

File: RELEASING.md (hunk 1 above)
Comment (bogdandrutu) [inline RELEASING.md]: upstream is the main repository. Usually origin refers to the personal fork and upstream to the remote main repository. Added a comment.

Comment (sebright) [inline RELEASING.md]: Thanks.  I read it again, and it's all clear now.

**Comment by adriancole** [comment]
Id recommend intermediating with bintray as the process is a lot less work
that way (bintray can do the signing etc and can api drive maven central
sync)

**Pull request merged**